<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetarsWindow</class>
 <widget class="QWidget" name="MetarsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>247</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Weather reports</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/imgs.qrc">
    <normaloff>:/mainIcon/icon.png</normaloff>:/mainIcon/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="MetarIcaoEdit">
       <property name="toolTip">
        <string>Type airport ICAO code here to fetch METAR</string>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
       <property name="placeholderText">
        <string>Enter an ICAO code here</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="FetchButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Download METAR</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/imgs.qrc">
         <normaloff>:/uiIcons/download.png</normaloff>:/uiIcons/download.png</iconset>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RefreshAllButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Refresh all</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/imgs.qrc">
         <normaloff>:/uiIcons/refresh.png</normaloff>:/uiIcons/refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClearButton">
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/imgs.qrc">
         <normaloff>:/uiIcons/delete.png</normaloff>:/uiIcons/delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ResizableListView" name="MetarListView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResizableListView</class>
   <extends>QListView</extends>
   <header>ui/views/resizablelistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/imgs.qrc"/>
 </resources>
 <connections/>
</ui>
